core/src/UI/AuthorizationResult.cs

```diff
@@ -87,6 +87,13 @@ internal AuthorizationResult(AuthorizationStatus status)
 
         public void ParseAuthorizeResponse(string webAuthenticationResult)
         {
+            if (!string.IsNullOrWhiteSpace(webAuthenticationResult) &&
+                webAuthenticationResult.StartsWith(Constants.FormPostPrefix, StringComparison.OrdinalIgnoreCase))
```

Comment:
>Constants.FormPostPrefix [](start = 51, length = 24)

minor: great that you added this one. for consistency we have one below msauth which could follow the same pattern?

Reply:
This code is not required anymore.